app/views/tags/index.html.erb

```diff
@@ -3,10 +3,16 @@
     Tags
   </div>
 
+  <% is_admin = @user.try(:is_admin?) %>
+  <% if is_admin %>
+    <div>
+      <%= link_to 'Create New Tag', new_tag_path %>
+    </div>
+  <% end %>
   <% max_size = @tags.map{|t| t.stories_count }.max %>
   <% @tags.each do |tag| %>
     <% mod = (max_size.to_f / ((tag.stories_count.to_f == 0) ? 1 : tag.stories_count.to_f )) %>
-    <%= link_to tag.tag, tag_path(tag), :class => tag.css_class,
+    <%= link_to tag.tag, is_admin ? edit_tag_path(tag.id) : tag_path(tag), :class => tag.css_class,
```

Comment:
Add an edit link, don't replace the link to the per-tag view.

Reply:
Trouble is, the all-tags display looks like this:

![image](https://user-images.githubusercontent.com/3979556/32152998-04d70b1a-bcfe-11e7-9e11-415db8619a6d.png)

So where do I put the edit link? An extra row of text in each bubble? I don't think we have a tag list view where it would look reasonable to add an extra link in.

Reply:
Ah, I misread and thought this template was for `/filters`, not `/tags`, my mistake. I don't have any better suggestion.